pkg/task/identity_test.go

```diff
@@ -24,7 +24,36 @@ import (
 func TestNewTaskIdentifier(t *testing.T) {}
 
 // TODO
-func TestIsPod(t *testing.T) {}
+func TestIsPod(t *testing.T) {
+	tests := map[string]struct {
+		taskIdentifier taskIdentifier
+		isPod          bool
+	}{
+		"True": {
+			taskIdentifier: taskIdentifier{
+				identity: MetaTaskIdentity{
+					Kind: "Pod",
```

Comment:
You can also put a test case having `Pod` in all lower case to indicate that the comparison is case sensitive or not.

Reply:
Agreed.